wiki/Ranking_Criteria/en.md

```diff
@@ -161,7 +161,7 @@ This category contains explicit allowance statements of concepts and rules that
 
 - **For songs where the composer(s) and singer(s) are different people, the singer(s) may be listed after the composer(s) or circle/group name following a `feat.` indicator.**
 - **If a Unicode song title or artist has an official translation or romanisation provided by the artist, it may be used in the respective romanised field. If both a translation and romanisation are available, either may be used.**
-- **If a beatmap's track was contributed to by multiple artists, they may be listed with commas in between.** If there are 3 or more contributing artists and they are not part of one officially labelled group, `Various Artists` or other descriptive artist labels may be used instead.
+- **If a beatmap's track was contributed to by multiple artists, they may be listed with commas in between.** If there are 3 or more contributing artists and they are not part of one officially labelled group, `Various Artists` or **other descriptive artist labels** may be used instead.
```

Comment:
all other bullet points have only the *first sentence* bolded. let's not break that format ty.

Reply:
how would you suggest going about this then? other bullet points with specific highlighting are because they link to another page, so they stand out

Reply:
italicising should do pretty much the same thing

```suggestion
- **If a beatmap's track was contributed to by multiple artists, they may be listed with commas in between.** If there are 3 or more contributing artists and they are not part of one officially labelled group, `Various Artists` or *other descriptive artist labels* may be used instead.
```

could also flip the sentence around to make that part more prominent

```suggestion
- **If a beatmap's track was contributed to by multiple artists, they may be listed with commas in between.** Either `Various Artists` or some other descriptive artist label may be used instead, if there are 3 or more contributing artists and they are not part of one officially labelled group.
```